### PR TITLE
Additional bugfix descriptions for 8.0.15-6 release

### PR DESCRIPTION
**Pull request by hrvojem**

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst [hunk 1]
```diff
@@ -43,12 +43,110 @@ New Features
...
+- upgrade from |Percona Server| 5.7.24 to :rn:`8.0.13-3` wasn't working with
+  encrypted undo tablespaces. Bug fixed :psbug:`5223`.
+
+- :ref:`keyring_vault_plugin` couldn't be initialized on *Ubuntu Cosmic 17.10*.
+  Bug fixed :psbug:`5453`.
+
+- rotated key encryption did not register ``encryption_key_id`` as a valid
+  table option. Bug fixed :psbug:`5482`.
+
+- :table:`INFORMATION_SCHEMA.GLOBAL_TEMPORARY_TABLES` queries may crash if
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst]: either `may ... is` or `might ... was`

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst [hunk 2]
```diff
@@ -43,12 +43,110 @@ New Features
...
+
+- ``FLUSH CHANGED_PAGE_BITMAPS`` would leave gaps between the last written
+  bitmap LSN and the |InnoDB| checkpoint LSN. Bug fixed :psbug:`5446`.
+
+- :ref:`changed_page_tracking` was missing pages changed by the in-place DDL.
+  Bug fixed :psbugs:`5447`.
+
+- ``innodb_system tablespace`` information was missing from the 
+  :table:`INFORMATION_SCHEMA.innodb_tablespaces` view.
+  Bug fixed :psbug:`5437`.
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst]: Is this the correct bug number?

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst [hunk 3]
```diff
@@ -43,12 +43,110 @@ New Features
...
+
+- undo log tablespace encryption status is now available through 
+  :table:`INFORMATION_SCHEMA.innodb_tablespaces` view.
+  Bug fixed :psbug:`5485` (upstream :mysqlbug:`94665`).
+
+- enabling temporay tablespace encryption didn't mark the 
+  ``innodb_temporary`` tablespace encryption flag. Bug fixed :psbug:`5490`.
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst]:
```suggestion
  ``innodb_temporary`` tablespace with encryption flag. Bug fixed :psbug:`5490`.
```

**Review by percona-ysorokin — APPROVED** [review]
LGTM

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst [hunk 4]
```diff
@@ -43,12 +43,111 @@ New Features
...
+- New :variable:`Audit_log_buffer_size_overflow` status variable has been
+  implemented to track when an :ref:`audit_log_plugin` entry was either
+  dropped or written directly to the file due to its size being bigger
+  than :variable:`audit_log_buffer_size` variable.
   
 
 Bugs Fixed
 ================================================================================
 
+- TokuDB and MyRocks native partitioning would never release memory which could
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst:56]: I don't think this is correct: the issue was that memory was allocated from a wrong memroot, where objects had wrong lifetime and also became corrupted on multithreaded access. @satya-bodapati please help with better description here

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst [hunk 5]
```diff
@@ -43,12 +43,111 @@ New Features
...
+  lead to a server crash. Bug fixed :psbug:`5508`.
+
+- using TokuDB or MyRocks native partitioning and ``index_merge`` could lead to
+  a server crash. Bugs fixed :psbug:`5206`, :psbug:`5562`.
+
+- upgrade from |Percona Server| 5.7.24 to :rn:`8.0.13-3` wasn't working with
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst:62]: https://jira.percona.com/browse/PS-5223 also discusses a workaround in the upgrade guide which should be removed with this fix, please check

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst [hunk 6]
```diff
@@ -43,12 +43,111 @@ New Features
...
+- :ref:`keyring_vault_plugin` couldn't be initialized on *Ubuntu Cosmic 17.10*.
+  Bug fixed :psbug:`5453`.
+
+- rotated key encryption did not register ``encryption_key_id`` as a valid
+  table option. Bug fixed :psbug:`5482`.
+
+- :table:`INFORMATION_SCHEMA.GLOBAL_TEMPORARY_TABLES` queries could crash if
+  online ``ALTER TABLE`` was running in parallel. Bug fixed :psbug:`5566`.
+
+- setting :variable:`log_slow_verbosity` and enabling the
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst:74]: settting to what?

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst [hunk 7]
```diff
@@ -43,12 +43,111 @@ New Features
...
+  :variable:`slow_query_log` could lead to a server crash. Bug fixed
+  :psbug:`4933`.
+
+- :ref:`compression_dictionary` operations were not handled under
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst:78]: Handled how, what is the effect of the fix?

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst [hunk 8]
```diff
@@ -43,12 +43,111 @@ New Features
...
+  intermediate binary log server in a replication chain. Bug fixed
+  :psbug:`5353`.
+
+- ``FLUSH CHANGED_PAGE_BITMAPS`` would leave gaps between the last written
+  bitmap LSN and the |InnoDB| checkpoint LSN. Bug fixed :psbug:`5446`.
+
+- :ref:`changed_page_tracking` was missing pages changed by the in-place DDL.
+  Bug fixed :psbug:`5447`.
+
+- ``innodb_system tablespace`` information was missing from the
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst:92]: ``innodb_system`` tablespace

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst [hunk 9]
```diff
@@ -43,12 +43,111 @@ New Features
...
+
+- long running ``ALTER TABLE ADD INDEX`` could cause a ``semaphore wait > 600``
+  assertion. Bug fixed :psbug:`3410` (upstream :mysqlbug:`82940`).
+
 - :ref:`backup_locks` was blocking DML for RocksDB. Bug fixed :psbug:`5583`.
 
+Other bugs fixed:
+:psbug:`5537`,
+:psbug:`5243`,
+:psbug:`5371`,
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst:130]: Either we publish the RHEL8 packages, either this one gets bumped to the next milestone. If bumped, then all RHEL8-related issues must get bumped

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst [hunk 10]
```diff
@@ -43,12 +43,111 @@ New Features
...
+:psbug:`5475`,
+:psbug:`5484`,
+:psbug:`5512`,
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst:133]: Either Disco / Buster packages are published, either this issue gets bumped to the next release

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst [hunk 11]
```diff
@@ -43,12 +43,111 @@ New Features
...
+:psbug:`5514`,
+:psbug:`5523`,
+:psbug:`5528`,
+:psbug:`5536`,
+:psbug:`5550`,
+:psbug:`5554`,
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst:139]: Needs description

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst [hunk 12]
```diff
@@ -43,12 +43,111 @@ New Features
...
+:psbug:`5570`,
+:psbug:`5578`,
+:psbug:`5441`,
+:psbug:`5442`,
+:psbug:`5456`,
+:psbug:`5462`,
+:psbug:`5487`,
+:psbug:`5489`,
+:psbug:`5501`,
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst:148]: A licence change, this one actually might need a description

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst (hunk 4 above)
Comment (satya-bodapati) [reply doc/source/release-notes/Percona-Server-8.0.15-6.rst:56]: Right, it is not only about ever growing memory. So I would suggest

TokuDB and MyRocks native partitioning handler objects allocated from wrong MEM_ROOT. Memory is released only on shutdown and concurrent access to global MEM_ROOT caused memory corruptions and therefore crashes. 

Feel free to reword if necessary.

Is this enough? Should we describe how we fixed? @laurynas-biveinis

Comment (laurynas-biveinis) [reply doc/source/release-notes/Percona-Server-8.0.15-6.rst:56]: @satya-bodapati LGTM. The focus in RNs should be in user terms

**Pull request merged**